extensions/positron-r/package.json

```diff
@@ -678,7 +683,9 @@
     "debuggers": [
       {
         "type": "ark",
-        "label": "R Debugger"
+        "label": "R Debugger",
+        "languages": ["r"],
+        "supportsUILaunch": false
```

Comment:
When we work on building Positron itself, the IDE registers the `vscode://schemas/vscode-extensions` schema at startup. This item is *not* in that schema, so if we go this direction overall, we will see a diagnostic in this file moving forward:

> Property supportsUILaunch is not allowed.

I did some exploration of how to silence this diagnostic but did not find anything quickly/easily.